libs/anglify/src/lib/modules/radio-button/radio-button.component.scss

```diff
@@ -0,0 +1,90 @@
+@use 'variables' as *;
+
+:host {
+  display: block;
+
+  .anglify-overlay__container {
+    height: $radio-button-ripple-height;
+    width: $radio-button-ripple-width;
+    margin: $radio-button-ripple-margin;
+    border-radius: $radio-button-ripple-border-radius;
+  }
+
+  .before {
+    flex-flow: row-reverse;
+  }
+
+  label {
+    display: flex;
+    position: relative;
+  }
+
+  input:not([disabled]) + .overlay-container {
+    cursor: pointer;
+  }
+
+  label .overlay-container {
+    width: $radio-button-ripple-inner-width;
+    height: $radio-button-ripple-inner-height;
+    display: flex;
+    overflow: visible !important;
+  }
+
+  label .overlay-container .radio-button-effect {
```

Comment:
Please add the property `box-sizing: border-box;` here. ✨

Explanation: https://developer.mozilla.org/de/docs/Web/CSS/box-sizing